code/controllers/stonedmc/subsystem/processing/projectiles.dm

```diff
@@ -0,0 +1,23 @@
+PROCESSING_SUBSYSTEM_DEF(projectiles)
+	name = "Projectiles"
+	wait = 1
+	stat_tag = "PP"
+	flags = SS_NO_INIT|SS_TICKER
+	var/global_max_tick_moves = 5
+	var/global_pixel_speed = 1
+	var/global_iterations_per_move = 16
+
+/datum/controller/subsystem/processing/projectiles/proc/set_pixel_speed(new_speed)
+	global_pixel_speed = new_speed
+	for(var/i in processing)
+		var/obj/item/projectile/P = i
+		if(istype(P))			//there's non projectiles on this too.
```

Comment:
Early continue

Reply:
`if(istype(P))
    do_stuff()`
vs
`if(!istype(P))
    continue
do_stuff()`

I don't get what you mean here.

Reply:
The second one